cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/type/ResourceEvent.java

```diff
@@ -3,8 +3,8 @@
 public enum ResourceEvent {
     WORKSPACE_CREATED("resource.workspace.created"),
     WORKSPACE_DELETED("resource.workspace.deleted"),
-    BLUEPRINT_CREATED("resource.blueprint.created"),
-    BLUEPRINT_DELETED("resource.blueprint.deleted"),
+    BLUEPRINT_CREATED("resource.clusterdefinition.created"),
```

Comment:
Should we rename the enums as well?

Reply:
There is a separated issue to rename public part